renderlib/io/FileReader.cpp

```diff
@@ -55,6 +55,13 @@ FileReader::getReader(const std::string& filepath, bool isImageSequence)
   } else if (extstr == ".zarr") {
     return new FileReaderZarr(filepath);
   }
+  // if it's a directory, and contains the string zarr anywhere, we assume it's a zarr
+  else if (std::filesystem::is_directory(filepath)) {
+    if (filepath.find("zarr") != std::string::npos) {
+      return new FileReaderZarr(filepath);
+    }
+  }
+  
```

Comment:
this change is a throw-in to allow loading zarrs from local directories when the directory name doesn't end in .zarr.  (Some conversion tools put the zarr in a subdirectory with a name like `my_file.zarr/0/`.